test/Chainweb/Test/Pact/PactSingleChainTest.hs

```diff
@@ -477,6 +482,19 @@ mempoolCreationTimeTest mpRefIO reqIO = testCase "mempoolCreationTimeTest" $ do
       unless (V.and oks) $ throwM $ userError "Insert failed"
       return txs
 
+preInsertCheckTimeoutTest :: IO (IORef MemPoolAccess) -> IO (PactQueue,TestBlockDb) -> TestTree
```

Comment:
https://github.com/kadena-io/chainweb-node/actions/runs/5693573278/job/15433182466?pr=1706
^ this test is flaky!

Reply:
Surprised it failed, runs okay locally. Set the timeout to 5 microseconds.

Reply:
Made the tx less nontrivial, works now.